src/components/Settings.vue

```diff
@@ -320,6 +320,27 @@
                           persistent-hint
                         ></v-switch>
                       </v-flex>
+                      <v-flex xs12 sm6>
+                        <v-switch
+                          hint="Enable logging"
+                          persistent-hint
+                          label="Log enabled"
+                          v-model="gateway.logEnabled"
+                        ></v-switch>
+                        <v-switch
```

Comment:
This should be visible just when logging is enabled

Reply:
done.